### ISSUE TITLE
Question: Recommendation for extending section "level" option?

### ISSUE THREAD

**Issue by james-cnz** [davidherney/moodle-format_onetopic#59]
Hi David,

I don't know how to message you, so I hope it's OK to ask here.

I've started writing a course format, with parent and child tabs for pages, like One-Topic Format, but also allowing multiple sections embedded in each page.  I thought it would make sense to use One-Topic format's section "level" option with 0 and 1 for parent and child tabs, and extend it with 2 for embedded sections?  Is this OK?

Thanks,
James

**Comment by davidherney** [comment]
Hi @james-cnz 

It can be some confusing for users but it is a possible use. However, can be a topic child and embedded in the same time? It is a decisive question.

Saludos

**Comment by james-cnz** [comment]
Hi @davidherney,

I'm not sure I understood your answer, or that I explained the question very well.  I think I've got my course format mostly working now, so I wonder if you have time to have a quick look at it, instead of me trying to explain it?  I want to make sure I don't create compatibility issues by extending the "level" option.

My course format's here if you've got time: https://github.com/james-cnz/moodle-format_multitopic

Thanks,
James.
